File: Unit 1-Into/Array Max/arrayMax.py
def maxOfArray(A, n):
    # A is array with size n
    if n >= 1:
        Result = A[0]
    else:
        Result = None
    for i in range(1, n):
        if A[i] > Result:
            Result = A[i]
    return Result

if __name__ == '__main__':
    array = [1,4,2,5,66,4,3,66,3,45,7]
    # array = []
    length = len(array)
    print(maxOfArray(array, length))

File: Unit 1-Into/Array Max/arrayMax.cpp
#include<iostream>
using namespace std;

int maxOfArray(int A[], int n)
{
    int result = A[0];
    for (int i=1; i<n; i++)
    {
        if (A[i] > result)
            result = A[i];
    }
    return result;
}

int main()
{
    int Array[] = {1,3,55,43,5,356,745,67,3};
    // int Array[] = {};
    int length = sizeof(Array)/sizeof(int);
    cout<<maxOfArray(Array, length);
    return 0;
}